profiles/factories.py

```diff
@@ -13,5 +13,7 @@ class ProfileFactory(DjangoModelFactory):
     image_small = Faker('file_path', extension='jpg')
     image_medium = Faker('file_path', extension='jpg')
 
+    email_optin = Faker('boolean')
```

Comment:
i also have a PR on MM to add email optin https://github.com/mitodl/micromasters/pull/3822